docs/releasing/after-publishing-a-release.md

```diff
@@ -0,0 +1,35 @@
+# Tasks after publishing a new release of GOV.UK Frontend
+
+Once we've published a new release of GOV.UK Frontend there are a number of things we still need to od such as informing our community and updating our own repos.
+
+## Update our repos to use the new release
+
+Developers should update [govuk-design-system](https://github.com/alphagov/govuk-design-system/issues/2024) and [govuk-frontend-docs](https://github.com/alphagov/govuk-frontend-docs) to use the new release. When updating the design system, you should also test any new guidance and/or guidance updates pertaining to the new release and publish them along with the version update.
+
+## Let the community know about the new release
+
+We now need to update or comms channels.
+
+### The mailing list
+
+The team should send out an email with the release comms drafted before publishing the release. This release should link to the associated github release on the [govuk-frontend releases page](https://github.com/alphagov/govuk-frontend/releases).
```

Comment:
Not sure the link here is right — [the emails typically link](https://mailchi.mp/4cd70c7f6c2e/new-release-govuk-frontend-v410-feature-release-10053738?e=[UNIQID]) to the release notes and npm how to.

Also suggesting a summary below.

Reply:
```suggestion
The team should send out an email with the release comms drafted before publishing the release. This release should link to the associated github release on the [govuk-frontend releases page](https://github.com/alphagov/govuk-frontend/releases).

The release email should:
- give a summary of the update
- explain on why teams should update
- components or styles affected (if possible)
- thanks to any major contributors (if possible, include name and organisation)

A release email will typically end with a link to the version's [release notes](https://github.com/alphagov/govuk-frontend/releases), and a call-to-action button to the [update npm how-to page](https://frontend.design-system.service.gov.uk/updating-with-npm/#update-using-node-js-package-manager-npm).

For example, see the [release email for GOV.UK Frontend 4.4.0](https://us1.admin.mailchimp.com/campaigns/show?id=10053738).
```

Reply:
Your suggestion covers this nicely. I'll absorb your suggestion and remove the reference to releases in my paragraph.